python/lsst/ctrl/mpexec/cli/script/report.py

```diff
@@ -24,34 +24,86 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import pprint
 
+import yaml
+from astropy.table import Table
 from lsst.daf.butler import Butler
```

Comment:
Probably should be a blank line between the astropy and Butler imports too, if you're being a format stickler.

Reply:
isort won't allow that.